cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,29 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+/// <reference types='cypress' />
```

Comment:
The `/// <reference types='cypress' />` declaration is redundant as it is already declared on line 1. Consider removing this duplicate reference to clean up the code.